src/controllers/sites-audits-toggle.js

```diff
@@ -81,6 +81,15 @@ export default (dataAccess) => {
             return { status: 404, message: `Site with baseURL: ${baseURL} not found.` };
           }
 
+          const validAuditTypes = Object.keys(configuration.getHandlers());
+          if (!validAuditTypes.includes(auditType)) {
+            return {
```

Comment:
you can use `notFound` utility function here instead of using the `Response` class directly